src/service/viewport/viewport-impl.js

```diff
@@ -425,20 +433,38 @@ export class ViewportImpl {
 
   /** @override */
   animateScrollIntoView(element, pos = 'top', opt_duration, opt_curve) {
-    devAssert(
-      !opt_curve || opt_duration !== undefined,
-      "Curve without duration doesn't make sense."
-    );
+    if (IS_SXG) {
+      const {promise, resolve} = new Deferred();
+      const waiter = debounce(
+        this.ampdoc.win,
+        () => {
+          this.ampdoc.win.removeEventListener('scroll', waiter);
+          resolve();
+        },
+        300
```

Comment:
can we move this constant somewhere? what does it represent?

Reply:
It's been moved.